docs/specifications/tests/Basic-TP/basic00.md

```diff
@@ -1,63 +1,314 @@
+A-label. A U-label must not contain any upper case letters.
+
+Special attention should be put on U+0049 (LATIN CAPITAL LETTER I) and U+0130
+(LATIN CAPITAL LETTER I WITH DOT ABOVE). Both must be mapped into U+0069 (LATIN
+SMALL LETTER I), also in a Turkish and Azeri locale. (See [Unicode SpecialCasing]
+for the special casing of U+0130 etc in Turkish and Azeri locale.) If a U-label
+contains a U+0131 (LATIN SMALL LETTER DOTLESS I) it will also be tested as such.
+
+#### Full stop
+
+The regular dot "." expected in domain names is a U+002E (FULL STOP). There are
+other characters that may be entered instead due to the script setting. The
+table 3 lists "full stops" can be mapped into the ASCII FULL STOP 
+([Unicode TR 46][Unicode TR 46#Notation], section 2.3). That mapping must be done
+before any verification or checks of the dot.
+
+*Table 3: Non-ASCII dots (Full Stops) using [Unicode] codes.*
+
+Unicode code | Character | Name
+:------------|:----------|:--------------------
+U+FF0E       | ．        | FULLWIDTH FULL STOP
+U+3002       | 。        | IDEOGRAPHIC FULL STOP
+U+FF61       | ｡         | HALFWIDTH IDEOGRAPHIC FULL STOP
+
+#### A-label and U-label
+
+DNS can only handle A-labels, not U-label. In the test core only A-labels are
+used. In Zonemaster, conversion from U-label to A-label is done at the perifery
+which can also convert back to U-labels even when A-label has originally be
+entered.
+
+## Scope
+
+This test case does not test the zone of *Child Zone*. It only tests and
+normalizes the name string.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+* "Valid ASCII" - Set of permitted ASCII characters in table 1 above.
+* "Full Stops" - Set of Full Stops in table 3 above with name Unicode code.
+* "Unicode" - The Unicode database.
+
+## Summary
+
+* This test case will not start if *Child zone* is empty.
+* If any message with severity level *[ERROR]* or *[CRITICAL]* is outputted, then
+  this test case will terminate all other test cases.
+* This test case will, potentially, normalize *Child Zone* and return that to
+  subsequent test cases.
+
+Message Tag outputted    | [Default level] | Description of when message tag is outputted
+:------------------------|:---------|:-----------------------------------------
+B00_NOT_UTF-8_ASCII      | CRITICAL | Domain name to be tested is encode in unknown character encoding.
+B00_INITIAL_DOT          | CRITICAL | Domain name starts with dot.
+B00_REPEATED_DOTS        | CRITICAL | Domain name has repeated dots.
+B00_INVALID_ASCII        | CRITICAL | ASCII label has characters not permitted.
+B00_INVALID_U_LABEL      | CRITICAL | Label with non-ASCII character is not valid U-label.
+B00_LABEL_TOO_LONG       | CRITICAL | Domain name has a label that is too long (more than 63 characters).
+B00_DOMAIN_NAME_TOO_LONG | CRITICAL | Domain name is too long (more than 253 characters with no final dot).
+
+
+## Ordered description of steps to be taken to execute the test case
```

Comment:
Is this section called "Test procedure" in the template? Should it be changed there or renamed here?

Reply:
In all specifications it is written as above. It is a very long headline, and after discussion with and suggestion from  @mattias-p I changed the secion headline in the proposed template. This is the first specification that I do any editing in after creating the template PR. I will look at some specification and adapt to the template.